javascripts/discourse/initializers/initialize-for-pdf-preview.js

```diff
@@ -16,7 +16,7 @@ export default {
         const previewModeSetting = settings.preview_mode;
         const newTabIcon = () => {
           const template = document.createElement("template");
-          template.innerHTML = iconHTML("external-link-alt", {
+          template.innerHTML = iconHTML("fa-solid fa-arrow-up-right-from-square", {
```

Comment:
```suggestion
          template.innerHTML = iconHTML("up-right-from-square", {
```
The solid style is applied by default in FA6, no need to add solid here. Our icon library also removes the `fa-` prefix so that prefix is unnecessary.  

`external-link-alt` maps to `up-right-from-square` in the mapping from old icon names https://github.com/discourse/discourse/blob/b195b1c8a61d208e698ff07d79d51cdbb6fa51bf/app/assets/javascripts/discourse/app/lib/icon-library.js#L279.